@xen-orchestra/web-core/lib/components/ui/button/UiButton.vue

```diff
@@ -0,0 +1,531 @@
+<!-- v4 -->
+<template>
+  <button :class="classNames" :disabled="busy || isDisabled" class="ui-button" type="button">
+    <VtsIcon :busy :icon="leftIcon" accent="current" class="icon" fixed-width />
+    <slot />
+    <VtsIcon :icon="rightIcon" accent="current" class="icon" fixed-width />
```

Comment:
According to the V2 UI button changelog, the right icon has been removed (unused).
It seems that if there is a right icon, it is always a "lock" icon.
And the prop name is: `rightIcon` -> `lockIcon`

Reply:
Maybe this prop could be a `boolean`, since the icon will always be the `faLock` one. And based on the examples from the DS, I think the button should be disabled when the prop value is set to `true`.